### PR TITLE
Add image augs

### PR DESCRIPTION
**Pull request by dpodvyaznikov**
Add transformations from PIL.ImageEnhance under single action `enhance_image`.
Remove `multiply_lightness` action.

### CODE REVIEW

File: batchflow/batch_image.py [hunk 1]
```diff
@@ -821,20 +821,38 @@ def _clip_(self, image, low=0, high=255):
...
+            'c': 'Contrast',
+            'b': 'Brightness',
+            's': 'Sharpness'
+        }
+
+        if len(order) != len(factors):
+            raise ValueError("'order' and 'factors' should be of same length!")
+
+        for alias, factor in zip(order, factors):
+            enhancement = enhancements.get(alias, None)
```
Comment (roman-kh) [inline batchflow/batch_image.py]:
```suggestion
            enhancement = enhancements.get(alias)
```

File: batchflow/batch_image.py [hunk 2]
```diff
@@ -821,20 +821,38 @@ def _clip_(self, image, low=0, high=255):
...
         ----------
-        multiplier : Number
-        src : str
-            Component to get images from. Default is 'images'.
-        dst : str
-            Component to write images to. Default is 'images'.
-        p : float
-            Probability of applying the transform. Default is 1.
+        layout : str
+            defines layout of operatoins, default is `hcbs`:
```
Comment (roman-kh) [inline batchflow/batch_image.py]:
```suggestion
            defines layout of operations, default is `hcbs`:
```

File: batchflow/batch_image.py [hunk 3]
```diff
@@ -821,20 +821,38 @@ def _clip_(self, image, low=0, high=255):
...
         """
-        return PIL.ImageEnhance.Brightness(image).enhance(multiplier)
+        enhancements = {
+            'h': 'Color',
+            'c': 'Contrast',
+            'b': 'Brightness',
+            's': 'Sharpness'
+        }
+
+        if len(layout) != len(factor):
```
Comment (roman-kh) [inline batchflow/batch_image.py]: This will fail if factor is float

Comment (dpodvyaznikov) [reply batchflow/batch_image.py]: Fixed

**Pull request merged**